### PR TITLE
Task completed: GP-117. Destroy by time

### PR DESCRIPTION
**Pull request by Kjusok**
https://lighteatersstudio.atlassian.net/jira/software/projects/GP/boards/1?selectedIssue=GP-117
Added Destroy projectile by time

### CODE REVIEW

File: Assets/Scripts/Gameplay/Projectile/Projectile.cs [hunk 1]
```diff
@@ -4,9 +4,12 @@ namespace Gameplay.Projectile
 {
     public class Projectile : MonoBehaviour
     {
+        private const float TimeForDestroyShot = 5;
```
Comment (ega-ega) [inline Assets/Scripts/Gameplay/Projectile/Projectile.cs]: pls change to serialized field

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Gameplay/Projectile/Projectile.cs (hunk 1 above)
Comment (Kjusok) [reply Assets/Scripts/Gameplay/Projectile/Projectile.cs]: Done

**Review by ega-ega — APPROVED** [review]

File: Assets/Scripts/Gameplay/Projectile/Projectile.cs [hunk 2]
```diff
@@ -26,11 +29,13 @@ public void Launch(Vector3 position, Vector3 direction, float damage, TypeDamage
 
         private void OnTriggerEnter(Collider other)
         {
+            var damageable = other.GetComponent<IDamageable>();
```
Comment (ega-ega) [inline Assets/Scripts/Gameplay/Projectile/Projectile.cs]: move this line under first if

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Gameplay/Projectile/Projectile.cs (hunk 2 above)
Comment (Kjusok) [reply Assets/Scripts/Gameplay/Projectile/Projectile.cs]: done

**Review by ega-ega — APPROVED** [review]

**Pull request merged**